share/spice/rain/rain.js

```diff
@@ -0,0 +1,34 @@
+function ddg_spice_rain (api_result) {
+    "use strict";
+
+    // Check for errors.
+    if(!api_result || api_result.error || !api_result.currently) {
```

Comment:
It might make sense to check for `item.flags['ddg-location']` as well just in case it's missing like in https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/forecast/forecast.js#L6...L9